src/shared/codelens/javaCodeLensProvider.ts

```diff
@@ -46,7 +44,7 @@ export async function getLambdaHandlerCandidates(document: vscode.TextDocument):
         return {
             filename: document.uri.fsPath,
             handlerName,
-            manifestUri: manifestUri,
+            rootUri: rootUri,
```

Comment:
Can Java lambdas work without a manifest file?
I know Python and Node-based lambdas don't need one, but not sure about Java.
Also, relaxing this requirement is going to make the lenses even noiser. Which we already have a problem with for Python.

Reply:
I'm re-scoping this to just JS/TS and Python. Your point is valid and they are the only types that are downloadable from the toolkit.

> Also, relaxing this requirement is going to make the lenses even noiser. Which we already have a problem with for Python.

Yes, this will potentially be noisier, but it solves the issue with downloaded lambdas. If it is too noisy, we have the option to toggle off codelenses.